examples/rbd/secret.yaml

```diff
@@ -13,3 +13,6 @@ stringData:
 
   # Encryption passphrase
   encryptionPassphrase: test_passphrase
+
+  # Vault token
```

Comment:
that is a pretty useless comment, it is exactly the same ad the name of the key... Either add value with a comment, or just do not add it at all. Here, you should probably mention it is only used when `vaultTokenAuth` is enabled.